### PR TITLE
Ci pipelines

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

File: samples/spring-boot-auto-config/.cache/formatter-maven-cache.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Fri Jan 21 13:29:12 CET 2022
```
Comment (robzienert) [inline samples/spring-boot-auto-config/.cache/formatter-maven-cache.properties:1]: Don't think these cache files should be getting commit.

**Review by robzienert — APPROVED** [review]
Aside from the cache dirs, looks good.

What formatter are you using? We use the Google Format internally, which is what I contributed with. I'd like to change my editor settings for this project so I can be consistent with the rest of this project in the future.

File: samples/spring-boot-auto-config/.cache/formatter-maven-cache.properties (hunk 1 above)
Comment (csviri) [reply samples/spring-boot-auto-config/.cache/formatter-maven-cache.properties:1]: removed, thx!

**Comment by csviri** [comment]
> Aside from the cache dirs, looks good.
> 
> What formatter are you using? We use the Google Format internally, which is what I contributed with. I'd like to change my editor settings for this project so I can be consistent with the rest of this project in the future.

Well, to be honest, since now the maven plugin formats the code, I don't even have it configured (used a google plugin before if remember correctly but wasn't working properly) since I always build the project before committing the source code.
Have some aliases like this instead :)
`alias mg='mvn clean install && git commit -a && git push' `

**Comment by robzienert** [comment]
That works, thanks. I don't use Maven normally, so I'm pretty out of my element when not using Gradle. :)

**Pull request merged**